MuchToDoWithoutAdo/app/src/main/java/com/scottlindley/muchtodowithoutado/Activities/ToDoListActivity.java

```diff
@@ -0,0 +1,210 @@
+package com.scottlindley.muchtodowithoutado.Activities;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.EditText;
+import android.widget.RadioButton;
+import android.widget.TextView;
+
+import com.scottlindley.muchtodowithoutado.JavaObjects.ToDo;
+import com.scottlindley.muchtodowithoutado.R;
+import com.scottlindley.muchtodowithoutado.RecyclerViewAdapters.ListScreenRecyclerViewAdapter;
+
+import static com.scottlindley.muchtodowithoutado.Activities.HomeActivity.mData;
+import static com.scottlindley.muchtodowithoutado.RecyclerViewAdapters.HomeRecyclerViewAdapter.POSITION_NUMBER;
```

Comment:
I describe this in more detail below, but it's not a good idea to static import specific variables from another class. It's less of an issue with constants, though even with those it would be more clear to simply refer to them by their qualified name (i.e. `HomeRecyclerViewAdapter.POSITION_NUMBER`) each time you reference it.